### PR TITLE
changed index.css for white text globally

### PR DESCRIPTION
**Pull request by dejazzhands**

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -18,5 +18,6 @@ module.exports = {
     }, 
     extend: {},
   },
-  plugins: [],
+  plugins: [
+  ],
```
Comment (ChristianBingman) [inline tailwind.config.js:22]: Could you get rid of this change

**Review by ChristianBingman — CHANGES_REQUESTED** [review]
Please remove the changes to package-lock and tailwind config

File: tailwind.config.js (hunk 1 above)
Comment (dejazzhands) [reply tailwind.config.js]: i deleted the file from commit. does this fix it?

Comment (ChristianBingman) [reply tailwind.config.js]: You didn’t delete it from the commit, you deleted it completely

**Pull request closed without merging**